src/ark/segmentation/marker_quantification.py

```diff
@@ -125,7 +126,7 @@ def assign_single_compartment_features(marker_counts, compartment, cell_props, c
     # add counts of each marker to appropriate column
     # Only include the marker_count features up to the last filtered feature.
     marker_counts.loc[compartment, cell_id,
-                      marker_counts.features[1]:filtered_regionprops_names[-1]] = cell_features
+    marker_counts.features[1]:filtered_regionprops_names[-1]] = cell_features
```

Comment:
Odd formatting here, although I'm surprised it works and the interpreter isn't throwing an error. We can probably leave it for now.